server/main/models.py

```diff
@@ -51,7 +51,17 @@ def get_id_url(self):
 class Followers(models.Model):
     author = models.ForeignKey(Author, related_name="followers", on_delete=models.CASCADE)
     followers = models.ManyToManyField(Author, related_name='author_followers')
-    
+
+    def friends(self):
+        list = []
+        for author in self.followers.all():
+            try:
+                object = Followers.objects.get(author=author)
+                if self.author in object.followers.all():
+                    list.append(author)
+            except Followers.DoesNotExist:
+                pass
+        return list
 
```

Comment:
Could we add another function is_friends()?
This is a rough idea, something like, and catch DoesNotExist too
```
def is_friends(self, author1, author2):
    check1 = Followers.objects.get(author=author1).followers.get(id=author2)
    check2 = Followers.objects.get(author=author2).followers.get(id=author1)
```